doc/api.md

```diff
@@ -0,0 +1,1326 @@
+
+```js
+{
+    "date_joined": "2020-01-14T22:35:49.958Z",
+    "id": 1,
+    "organization": "Example Organization",
+    "last_login": null,
+    "is_staff": false,
+    "is_superuser": false,
+    "email": "info@example.com",
+    "thumbnail": null,
+    "is_active": true,
+    "status": "success",
+    "photo": null,
+    "allow_email_contact": false,
+    "last_name": "Person",
+    "first_name": "Demo",
+    "make_info_public": false,
+    "username": "demoperson"
+}
+```
+
+---
+
+`PUT /user/{id}`
+
+Update the profile details of a user. A user may update their profile and only their profile.
+
+Returns a the full, updated profile details of the specified user as a JSON object.
+
+```js
+{
+    "date_joined": "2020-01-14T22:35:49.958Z",
+    "id": 1,
+    "organization": "Example Organization",
+    "last_login": null,
+    "is_staff": false,
+    "is_superuser": false,
+    "email": "info@example.com",
+    "thumbnail": null,
+    "is_active": true,
+    "status": "success",
+    "photo": null,
+    "allow_email_contact": false,
+    "last_name": "Person",
+    "first_name": "Demo",
+    "make_info_public": false,
+    "username": "demoperson"
+}
+```
+
+---
+
+`POST /user/{id}/photo`
+
+Upload a profile photo for a user.
+
+Returns a JSON object.
+
+```js
+{
+    "url": "https://photopath/thumbnail.jpg"
+}
+```
+
+---
+
+`POST /send-password-reset-email`
+
+Start the password reset process by sending an email.
+
+Query parameters:
+
+- `email` (required) - The email address of a registered user.
+
+Returns a JSON object
+
+```js
+{
+    "status": "success"
+}
+```
+
+### Instance
+
+`GET /instances`
+
+Get a list of publicly accessible instances.
+
+Returns an array of JSON objects
+
+```js
+[
+    {
+        "center": {
+            "lng": -112.12762796884756,
+            "lat": 33.58432998116328
+        },
+        "id": 2,
+        "url": "apidemo",
+        "universalRevHash": "c81e728d9d4c2f636f067f89cc14862c",
+        "name": "API Demo",
+        "extent_radius": 70710.67811865476,
+        "extent": {
+            "max_lng": -111.67847032678779,
+            "max_lat": 33.957698113234656,
+            "min_lng": -112.57678561090732,
+            "min_lat": 33.209339275354175
+        },
+        "eco": {
+            "supportsEcoBenefits": true,
+            "benefits": [
+                {
+                    "label": "Tree Benefits",
+                    "model": "plot",
+                    "keys": [
+                        "energy",
+                        "stormwater",
+                        "airquality",
+                        "co2",
+                        "co2storage"
+                    ]
+                }
+            ]
+        },
+        "geoRevHash": "c81e728d9d4c2f636f067f89cc14862c"
+    }
+]
+```
+---
+
+`GET /locations/{latitude},{longitude}/instances`
+
+Get the instances which the current user has contributed to and instances within a specified distance of the specified point.
```

Comment:
This mentions a "specified distance" - but that doesn't appear to be a path or query parameter. Can this distance actually be specified or is it hard coded?

Reply:
Good catch. There is a `distance` query string param. Added in fixup c79d39c7.